en/docs/deploy/identity-related-tables.md

```diff
@@ -435,9 +434,9 @@ description of the values it contains.
 
 #### IDN_ASSOCIATED_ID
 
-In the Identity Server Dashboard, users can associate their social
+In WSO2 Identity Server Dashboard, users can associate their social
```

Comment:
`Dashboard` is outdated

Reply:
Commented it out.

Reply:
ideally, this shouldn't be commented out. This functionality has been there in the dashboard but since now we have my account as its successor please check if the functionality is there and update it here accordingly.

Reply:
You can only view the social logins that you've already used in **My Account**. Anyway, added it back and removed the dashboard part since the table still exists.